### PR TITLE
Rough skeleton complete

### PR DESCRIPTION
**Pull request by jfleming9357**

### CODE REVIEW

**Comment by jfleming9357** [comment]
@daniel-he-dev @cayla-c @mattBman23

**Review by cayla-c — APPROVED** [review]
Looks good!

File: client/components/RatingsReviews/ReviewList.jsx [hunk 1]
```diff
@@ -14,19 +15,31 @@ const getReviews = (pId) => {
 export const ReviewList = (props) => {
   const [loading, setLoading] = useState(true);
```
Comment (daniel-he-dev) [inline client/components/RatingsReviews/ReviewList.jsx:16]: use of loading state variable seems unnecessary. although this code is fine, maybe consider adding a small tech debt ticket to refactor to useEffect hooks? These accomplish similar things to class lifecycle methods.

**Comment by daniel-he-dev** [comment]
Had one comment, but go ahead and merge

**Pull request merged**